pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "temporalio"
-version = "0.1a2"
+version = "0.1b1"
```

Comment:
Kind of curious why we don't use semver - is it not widely used in the Python world? FWIW, I think most of the promises of semver are BS but at least it has a formalized syntax

Reply:
We will use semver upon GA. This is Python's knowingly-non-semver form of expressing alpha/beta. See https://peps.python.org/pep-0440/ (specifically https://peps.python.org/pep-0440/#semantic-versioning and discussion therein about prerelease builds).

Reply:
Cooll, thanks for the link.